lib/android-helpers.js

```diff
@@ -476,22 +476,34 @@ helpers.setMockLocationApp = async function setMockLocationApp (adb, app) {
   }
 };
 
-helpers.installHelperApp = async function installHelperApp (adb, apkPath, packageId, appName) {
-  try {
+helpers.installHelperApp = async function installHelperApp (adb, apkPath, packageId) {
+  // Rarely adb push or adb install takes much time to do them:
```

Comment:
Sometimes `adb push` or `adb instal` take more time than expected to install an app